src/hotspot/share/cds/cdsConfig.cpp

```diff
@@ -406,6 +406,10 @@ void CDSConfig::check_aot_flags() {
   CHECK_SINGLE_PATH(AOTCacheOutput);
   CHECK_SINGLE_PATH(AOTConfiguration);
 
+  if (FLAG_IS_DEFAULT(AOTCache) && AOTAdapterCaching) {
+    log_debug(aot,codecache,init)("AOTCache is not specified - AOTAdapterCaching is ignored");
+  }
+
```

Comment:
Also from merge?

Reply:
That's from the merge. This block doesn't show up when you select "show all commits".